commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountClayTable.java

```diff
@@ -107,22 +104,57 @@
 				LanguageUtil.get(httpServletRequest, "view"), false, false);
 
 			clayTableActions.add(clayTableViewAction);
+
+			CommerceContext commerceContext =
+				(CommerceContext)httpServletRequest.getAttribute(
+					CommerceWebKeys.COMMERCE_CONTEXT);
+
+			CommerceAccount currentCommerceAccount =
+				commerceContext.getCommerceAccount();
+
+			if (account.getAccountId() !=
+					currentCommerceAccount.getCommerceAccountId()) {
+
+				StringBundler sb = new StringBundler(7);
+
+				sb.append("javascript:setCurrentAccount");
+				sb.append(StringPool.OPEN_PARENTHESIS);
+				sb.append(StringPool.APOSTROPHE);
+				sb.append(account.getAccountId());
+				sb.append(StringPool.APOSTROPHE);
+				sb.append(StringPool.CLOSE_PARENTHESIS);
+				sb.append(StringPool.SEMICOLON);
+
+				ClayTableAction clayTableSetActiveAction = new ClayTableAction(
+					StringPool.BLANK, sb.toString(), StringPool.BLANK,
+					LanguageUtil.get(httpServletRequest, "select"), false,
+					false);
+
+				clayTableActions.add(clayTableSetActiveAction);
+			}
 		}
 
 		if (_modelResourcePermission.contains(
 				themeDisplay.getPermissionChecker(), account.getAccountId(),
 				ActionKeys.UPDATE)) {
 
-			String setActiveURL = _getAccountSetActiveURL(
-				account.getAccountId(), httpServletRequest);
+			StringBundler sb = new StringBundler(7);
+
+			sb.append("javascript:toggleActiveCommerceAccount");
+			sb.append(StringPool.OPEN_PARENTHESIS);
+			sb.append(StringPool.APOSTROPHE);
```

Comment:
same here.

Reply:
for record: https://issues.liferay.com/browse/COMMERCE-2354